src/libs/StepperMotor.cpp

```diff
@@ -29,6 +35,7 @@ StepperMotor::StepperMotor(Pin &step, Pin &dir, Pin &en) : step_pin(step), dir_p
     acceleration= NAN;
     selected= true;
     extruder= false;
+    enable_delay= THEKERNEL->config->value(stepper_driver_enable_delay_checksum)->by_default("0.0")->as_number();
```

Comment:
Stepper motors are not modules and therefore are not allowed to read the config or set stuff from the config.